<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>component_order</class>
 <widget class="QMainWindow" name="component_order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>商家地址</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>顾客地址</string>
    </property>
   </widget>
   <widget class="QLabel" name="restaurant_loc">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>261</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>暂无</string>
    </property>
   </widget>
   <widget class="QLabel" name="customer_loc">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>261</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>暂无</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>订单编号</string>
    </property>
   </widget>
   <widget class="QLabel" name="order_id">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>261</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>暂无</string>
    </property>
   </widget>
   <widget class="QPushButton" name="have_finished">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>完成</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
